stories/exercises/05-customize-theme-tokens.mdx

```diff
@@ -0,0 +1,123 @@
+import { Meta, ColorPalette, ColorItem } from "@storybook/blocks";
+
+# 05. Customize with theme tokens
+
+With content on our landing page, we're going to make sweeping changes using USWDS Design Tokens and theme settings.
+
+You can view all available theme settings by visiting the [USWDS Settings Table](https://designsystem.digital.gov/documentation/settings/#general-settings-2).
+
+Lets start by updating some of our [theme color tokens](https://designsystem.digital.gov/documentation/settings/#color-settings-2). USWDS theme color tokens are designed to be useful and effective for any project size.
```

Comment:
**polish:** Swapping sentences could make it a good intro.

```suggestion
USWDS theme color tokens are designed to be useful and effective for any project size. Let's start by updating some of our [theme color tokens](https://designsystem.digital.gov/documentation/settings/#color-settings-2).
```

Reply:
Resolved in [360be53](https://github.com/Bixal/storybook-rapid-prototyping/pull/13/commits/360be53c2a7e2312574226e332e9e991700370f2)